src/Controller/PageAdminController.php

```diff
@@ -90,9 +92,9 @@ public function composeContainerShowAction(Request $request = null): Response
         $blockServices = $this->blockServiceManager->getServicesByContext('sonata_page_bundle', false);
 
         // Filter service using the template configuration
-        if ($page = $block->getPage()) {
+        if ($page = $block->getPage()) { // @phpstan-ignore-line
```

Comment:
what's the error here?

Reply:
PHPStan can't call this method, it remains undefined

Reply:
I think previous check `if (!$block) {` should be updated to fix that